tools/dynamic/body_tags.py

```
@@ -0,0 +1,31 @@
+            if in_tags:
+                continue
+            continue
+        if stripped.startswith("tags:"):
+            in_tags = True
+            value = stripped.split(":", 1)[1].strip()
+            if value.startswith("[") and value.endswith("]"):
+                tags.extend(clean_tag(part) for part in value[1:-1].split(","))
+            elif value and value not in {"[]", ""}:
+                tags.append(clean_tag(value))
+            continue
+        if in_tags and (line.startswith(" ") or line.startswith("\t")) and stripped.startswith("-"):
```

Ruff (0.15.15)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)